legacy/src/test/java/org/opensearch/sql/legacy/unittest/utils/BackticksUnquoterTest.java

```diff
@@ -23,20 +23,25 @@ public class BackticksUnquoterTest {
   public void assertNotQuotedStringShouldKeepTheSame() {
     assertThat(unquoteSingleField("identifier"), equalTo("identifier"));
     assertThat(unquoteFullColumn("identifier"), equalTo("identifier"));
+    assertThat(unquoteFullColumn(".identifier"), equalTo(".identifier"));
   }
 
   @Test
   public void assertStringWithOneBackTickShouldKeepTheSame() {
     assertThat(unquoteSingleField("`identifier"), equalTo("`identifier"));
     assertThat(unquoteFullColumn("`identifier"), equalTo("`identifier"));
+    assertThat(unquoteFullColumn("`.identifier"), equalTo("`.identifier"));
   }
 
   @Test
   public void assertBackticksQuotedStringShouldBeUnquoted() {
     assertThat("identifier", equalTo(unquoteSingleField("`identifier`")));
+    assertThat(".identifier", equalTo(unquoteSingleField("`.identifier`")));
 
     assertThat(
         "identifier1.identifier2", equalTo(unquoteFullColumn("`identifier1`.`identifier2`")));
     assertThat("identifier1.identifier2", equalTo(unquoteFullColumn("`identifier1`.identifier2")));
+    assertThat(
+        ".identifier1.identifier2", equalTo(unquoteFullColumn("`.identifier1`.`identifier2`")));
```

Comment:
Does the queries (without join) work before this fixing?
```sql
select q.query_id 
from .ubi_log_queries q

select q.query_id 
from `.ubi_log_queries` q
```
Could you add some tests to verify the objective issue is fixed:
```sql
select q.query_id 
from .ubi_log_queries q join .ubi_log_events e
on q.query_id = e.query_id

select q.query_id 
from `.ubi_log_queries` q join `.ubi_log_events` e
on q.query_id = e.query_id
```

Reply:
Yes, those queries work fine.

I'm open to testing suggestions, but the issue is the presence of the period inside the backticks. When an index starts with a `.`, the period is not included in the backticks.